src/components/MarkdownProvider/components/CodeExample/index.tsx

```diff
@@ -46,6 +46,13 @@ export const CodeExample: React.FC<{ code: string }> = ({ children, code }) => {
           css={css`
             padding: ${p => p.theme.space.md};
             direction: ${p => p.theme.rtl && 'rtl'};
+            overflow: auto;
+
+            &::after {
+              display: inline-block;
+              width: ${props => props.theme.space.base * 3}px;
+              content: '';
+            }
```

Comment:
This doesn't seem to be working as suggested. Let's just remove for now and address later.